### DISCUSSION TITLE
图录和表录统计不全

### DISCUSSION THREAD

**Discussion by Bennett561** [sjtug/SJTUThesis#741] [提问]
文章中插了不少图了，但是插图索引中只统计到一个，想知道这是为啥呀
note: 看了一下，插图用的代码都是统一的格式
```
\begin{figure}[!htp]
  \centering
  \includegraphics[width=6cm]{figures/VI.pdf} \\
  \bicaption[]
    {有效不等式示意图(红色虚线)}
    {Diagram of the valid inequality (red dashed line)}
 \label{fig:VI}
\end{figure}
```

![Snipaste_2021-11-29_20-58-16](https://user-images.githubusercontent.com/17747051/143872587-f17ca83c-0cae-441a-b597-91e551cd1396.png)

**Comment by skyzh** [comment]
每个 figure 的 label 需要不同。

**Reply by skyzh** [reply]
如果不是这个问题，请提供编译 log？

**Reply by Bennett561** [reply]
手动检查了label，好像没发现重复的，compile脚本怎么保留log啊？好像自动删除了

**Reply by Bennett561** [reply]
[log.txt](https://github.com/sjtug/SJTUThesis/files/7618901/log.txt)
搞到了

**Comment by WangXuhongCN** [comment]
不要使用\bicaption[]，而是直接使用\bicaption

**Reply by Bennett561** [reply]
soga~
